src/types.d.ts

```diff
@@ -94,6 +94,13 @@ declare global {
   type EnterpriseFeatures = {
     catalogQuerySearchFiltersEnabled?: boolean,
   };
+
+  export type LearnerProfileType = {
+    subscriptions: string[],
+    group_memberships: string[],
+    enrollments: string[],
+  };
```

Comment:
I don't have the full payload in front of me, but from what I do know it should look something like this:
```suggestion
  // 
  export type ErrorSection = { error: string };
  export type LearnerProfileType = {
    subscriptions: LearnerProfileSubscriptionTBD[] | ErrorSection,
    group_memberships: GroupMembershipTBD[] | ErrorSection,
    enrollments: {
         "in_progress": EnrollmentTBD[],
        "upcoming": EnrollmentTBD[],
        "completed": EnrollmentTBD[],
        "saved_for_later": EnrollmentTBD[]
    } | ErrorSection,
  };
```